snuba/datasets/schema.py

```diff
@@ -118,3 +126,62 @@ def __init__(self, local_table_name, dist_table_name, columns,
 
     def _get_engine_type(self):
         return "ReplacingMergeTree(%s)" % self.__version_column
+
+
+class SummingMergeTreeSchema(MergeTreeSchema):
+
+    def _get_engine_type(self):
+        return "SummingMergeTree()"
+
+
+class MaterializedViewSchema(Schema):
+
+    def __init__(
+            self,
+            local_table_name,
+            dist_table_name,
+            columns,
+            query,
+            local_src_table_name,
+            dist_src_table_name,
+            local_dest_table_name,
+            dist_dest_table_name
```

Comment:
Having dist_table_name and dest_table_name in the same function will eventually break. I would change the naming.

Reply:
Oh interesting, why's that?

Reply:
because it is so easy to swap them by a one letter typo during any change.